src/databricks/labs/ucx/assessment/pipelines.py

```diff
@@ -50,6 +50,16 @@ def _assess_pipelines(self, all_pipelines) -> Iterable[PipelineInfo]:
             pipeline_config = pipeline_response.spec.configuration
             if pipeline_config:
                 failures.extend(self.check_spark_conf(pipeline_config, "pipeline"))
+            pipeline_cluster = pipeline_response.spec.clusters
+            if pipeline_cluster:
```

Comment:
If not ...: continue 

How many times do I need to repeat it?

Reply:
@nfx . Here we can't exit from the loop as we are checking the failure list.This is similar to pipeline_config.